.github/workflows/aws_openshift_rosa_hcp_single_region_tests.yml

```diff
@@ -443,12 +469,14 @@ jobs:
                   secrets: ${{ toJSON(secrets) }}
                   camunda-version: ${{ steps.camunda-version.outputs.CAMUNDA_VERSION }}
                   camunda-domain: ${{ env.CAMUNDA_DOMAIN }}
+                  camunda-domain-grpc: zeebe-${{ env.DOMAIN_NAME }}:443
```

Comment:
This will be breaking your tests.

For EKS I handled it similar to `env.CAMUNDA_DOMAIN` and exported it in the same if statement as otherwise I was overwriting the default and then the test will run against `zeebe-:443`.

So in case of `domain` it's defined otherwise empty.